src/lib.rs

```diff
@@ -2228,6 +2230,16 @@ pub extern "cdecl" fn is_likely_desynced() -> bool {
     unsafe { LIKELY_DESYNCED }
 }
 
+#[no_mangle]
+pub extern "cdecl" fn get_ping() -> *mut i32 {
+    unsafe {
+        match NEXT_DRAW_PING {
+            Some(x) => Box::into_raw(Box::new(x)),
+            None => std::ptr::null_mut(),
+        }
+    }
+}
+
```

Comment:
When the function returns, the allocated `Box` has been disowned and freed.
An alternative way is returning in `i32` and using `-1` for the situation that `NEXT_DRAW_PING` is `None`.

Reply:
Oh sorry I made a mistake. Just ignore my previous comment.

Reply:
Well but seems that it can cause memory leak? And it'd be better to avoid allocating in heap when possible.